### PR TITLE
Lab color space

### PR DESCRIPTION
**Pull request by peterjiz**

### CODE REVIEW

**Comment by peterjiz** [comment]
Sure! Soon

File: kornia/color/lab.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        torch.tensor: LAB version of the image.
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+    Examples::
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> lab = kornia.color.RgbToLab()
+        >>> output = lab(input)  # 2x3x4x5
+    Reference::
+        [1]
```
Comment (ducha-aiki) [inline kornia/color/lab.py]: Reference?

File: kornia/color/lab.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+    def forward(  # type: ignore
+            self, input: torch.Tensor) -> torch.Tensor:
+        return rgb_to_lab(input)
+
+    
+def srgb_to_xyz(input: torch.Tensor) -> torch.Tensor:
+    srgb_pixels = input
+    
+    # Constant tensor used for conversion
```
Comment (ducha-aiki) [inline kornia/color/lab.py]: Reference for conversion  matrix?

Comment (edgarriba) [reply kornia/color/lab.py]: agree. lets block this one until tests are provided

**Comment by peterjiz** [comment]
Sorry for the delays. Added color conversion tests. 
Also the implementation is a port from a tensorflow implementation. I wrote other non-committed  tests comparing it with opencv

**Comment by edgarriba** [comment]
@peterjiz why do you have 202 modified files ? XD make sure you rebase the code in the proper way

**Comment by edgarriba** [comment]
@peterjiz could you try to rebase from master to solve the massive touched files issue ? we plan a release this coming week and this definitely should be in :)

**Comment by peterjiz** [comment]
> @peterjiz could you try to rebase from master to solve the massive touched files issue ? we plan a release this coming week and this definitely should be in :)

That would be awesome! I'll try! The reason why many files were touched is because I've directly incorporated kornia into my project, and two-way sync between my local machine and gpu server.

**Comment by edgarriba** [comment]
@peterjiz did you have the time to test this ? I have been tryng my own implementation for rgb_to_lab and compared to opencv and doesn't seem to produce same results.

I leave here a colab with basic code: https://colab.research.google.com/drive/1D8FqfIz7jpyzCHJmgEThHHoO2PnsJ1Fc

**Comment by stefanopini** [comment]
@peterjiz @edgarriba Any update on this?

**Comment by edgarriba** [comment]
@stefanopini this PR needs some refactoring before being merged. Anyone is welcomed to continue this.

**Comment by cceyda** [comment]
If by refactoring you mean a rebase with the latest changes from master, I can work on that.

**Comment by edgarriba** [comment]
@cceyda see my comment in your PR :)

**Pull request closed without merging**